feature-libs/cart/main/components/cart/clear-cart/clear-cart.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
+import { ActiveCartFacade } from '@spartacus/cart/main/root';
+import { take } from 'rxjs/operators';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ClearCartService {
+  isClearing$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+
+  constructor(
+    protected globalMessageService: GlobalMessageService,
+    protected activeCartService: ActiveCartFacade
```

Comment:
Le's name the var `activeCartFacade` instead.   I recognize that there might still be a few occurrences of `activeCartService: ActiveCartFacade` in the codebase as a result of recently changing the type from ActiveCartService to ActiveCartFacade.

Reply:
Will do